ts/Extensions/ExportData.ts

```diff
@@ -51,6 +52,7 @@ const {
 
 declare module '../Core/Chart/ChartLike'{
     interface ChartLike {
+        ascendingOrderInTable?: boolean
```

Comment:
It would be nice to avoid more decoration on the Chart, but okay if we have no better way to put it. In the future we probably need to make a composition for all exporting and export-data stuff.